libwidgets/View/SearchResults.vala

```diff
@@ -67,15 +67,20 @@ namespace Files.View.Chrome {
                         path_string: path_string,
                         file: parent.resolve_relative_path (info.get_name ()),
                         sortkey: category.to_string () + _name);
+
             }
 
             public Match.from_bookmark (Bookmark bookmark, SearchResults.Category category) {
-                Object (name: Markup.escape_text (bookmark.basename),
+                var _name = bookmark.custom_name != "" && bookmark.custom_name != bookmark.basename ?
+                _("%s (%s)").printf (bookmark.custom_name, bookmark.basename) :
+                bookmark.basename;
```

Comment:
Small style thing, should these lines be implemented to indicate they're continuing the `var _name` line?
```suggestion
                    _("%s (%s)").printf (bookmark.custom_name, bookmark.basename) :
                    bookmark.basename;
```